exercises/clock/canonical-data.json

```diff
@@ -1,9 +1,9 @@
 {
   "exercise": "clock",
-  "version": "2.2.0",
+  "version": "2.2.1",
   "comments": [
     "Most languages require constructing a clock with initial values,",
-    "adding a positive or negative number of minutes, and testing equality",
+    "adding or subtracting some number of minutes, and testing equality",
     "in some language-native way.  Some languages require separate add and",
```

Comment:
> "Some languages require separate add and subtract functions."

I am not sure that this sentence is really necessary.  Opinions?

Reply:
I agree that that sentence is not really necessary.

Reply:
in the previous state, it contrasts with the previous sentence's approach of having only `add` but having add support both +ve and -ve. In the current state it doesn't add more information since the previous sentence already has both add and subtract

because of this I think it makes more sense to remove it

Reply:
Thanks guys, I'll work on removing it.